spec/features/admin_node_update_feature_spec.rb

```diff
@@ -41,20 +41,30 @@
     end
     # rubocop:enable RSpec/MultipleExpectations
 
+    # rubocop:disable RSpec/ExampleLength
     scenario "User clicks on 'Reboot to update'", js: true do
-      allow(::Velum::Salt).to receive(:call)
+      allow(::Velum::Salt).to receive(:call).and_return(true)
 
       # clicks on "Update admin node"
       find(".update-admin-btn").click
 
+      # wait modal to appear
+      wait_until { page.has_text?("Reboot to update") }
+
       # clicks on "Reboot to update"
       find(".reboot-update-btn").click
 
-      # rubocop:disable RSpec/MessageSpies
-      expect(::Velum::Salt).to receive(:call)
-      # rubocop:enable RSpec/MessageSpies
+      wait_until { page.has_text?("Rebooting...") }
```

Comment:
Moreover, I see that the default is `default_max_wait_time`, which in our case is 5. So raising that to 10 would have the same effect. Except that what if it needs more than 10 seconds ? Should we increase this value to 60 or 120 just to be sure ? Does it have any side-effects doing so ?